frontend/build-configs/bayern/index.ts

```diff
@@ -104,11 +104,13 @@ const bayern: BuildConfigType = {
             excludeLocationPlayServices: false,
             excludeX86: false,
         },
+        appStoreLink: `https://play.google.com/store/apps/details?id=${ANDROID_APPLICATION_ID}`,
     },
     ios: {
         ...bayernCommon,
         bundleIdentifier: IOS_BUNDLE_IDENTIFIER,
         provisioningProfileSpecifier: "match AppStore de.nrw.it.ehrensachebayern",
+        appStoreLink: "https://apps.apple.com/de/app/ehrenamtskarte-bayern/id1261285110",
```

Comment:
:upside_down_face: Optional: Personally I think it might be better to also just add the `appStoreId` to the build configs, e.g. `id1261285110` in this case.
This would
- allow us to use the appStoreId separately for e.g. an `apple-itunes-app` meta tag for websites
- allow us to set the language of the link dynamically based on the users language
- reduce the chance of copy/paste mistakes like setting the language to `ky`

However, due neither of the first two points being relevant to us atm, feel free to just keep this as it is.

Reply:
hm same as i mentioned above. the i also need this`ehrenamtskarte-bayern` separately.
I don't see a big improvement here and will keep my solution